Foundations/FlexBox/index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Header Profile</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="index.css">
+  </head>
+
+  <body>
+          <div class="content">
```

Comment:
fix the html structure

`nav`
`main`
`footer`

Reply:
I updated my file and added new tags of nav, header. Please check